VisualizationBase/src/nodes/ViewItemNode.h

```diff
@@ -58,7 +62,13 @@ class VISUALIZATIONBASE_API ViewItemNode : public Super<UINode>
 		void setSpacingParent(ViewItemNode* spacingParent);
 		ViewItemNode* spacingParent() const;
 
+		virtual QJsonValue toJson() const;
+
 	private:
+		ViewItemNode(const ViewItem* parent);
+
+		const ViewItem* parent_{};
```

Comment:
Yeah, that needs to be thought through if we ever go that way. My initial idea was not to have a VViewItemNode at all. But we'll leave this for another time.